src/components/Editor/Tabs.css

```diff
@@ -108,13 +105,9 @@
 }
 
 .source-tab .close-btn {
```

Comment:
same here, could we merge this with the rule in https://github.com/devtools-html/debugger.html/pull/1976/files#diff-d9abd0398fe2e0fa5793ba7d100231f5R91 ? If I'm correct they are the same